packages/components/src/RichTextEditor/utils/core/createRichTextEditor.ts

```diff
@@ -58,9 +58,7 @@ export const createRichTextEditor = ({
   })
 
   return {
-    destroy: () => {
-      editorView && editorView.destroy()
-    },
+    destroy: () => editorView.destroy(),
```

Comment:
This looks like it checks out. Just to regurgitate my understanding: because we're assigning a value to editorView on ln 53, we don't (and guess we never did) need the `&&` so we wont need  `editorView?.destroy()`

Reply:
No idea. I just removed the first part and it didn't complain about the second, so I left it 😅 I can put the question mark back in if you're worried the types are actually inaccurate?

Reply:
I mean it makes sense based on the code thats there so if types aren't complaining that should be pretty safe